lnst/Recipes/ENRT/GeneveTunnelRecipe.py

```diff
@@ -128,17 +132,16 @@ def create_tunnel(self, configuration):
         # A
         m1.gnv_tunnel.mtu = 1400
         m1.gnv_tunnel.up_and_wait()
-        m1.gnv_tunnel.ip_add(a_ip4)
-        m1.gnv_tunnel.ip_add(a_ip6)
+        config.configure_and_register_ip(m1.gnv_tunnel, a_ip4)
+        config.configure_and_register_ip(m1.gnv_tunnel, a_ip6)
 
         # B
         m2.gnv_tunnel.mtu = 1400
         m2.gnv_tunnel.up_and_wait()
-        m2.gnv_tunnel.ip_add(b_ip4)
-        m2.gnv_tunnel.ip_add(b_ip6)
+        config.configure_and_register_ip(m2.gnv_tunnel, b_ip4)
+        config.configure_and_register_ip(m2.gnv_tunnel, b_ip6)
 
-        configuration.tunnel_devices.extend([m1.gnv_tunnel, m2.gnv_tunnel])
-        self.wait_tentative_ips(configuration.tunnel_devices)
```

Comment:
You removed this from the code, so we will now won't wait for tentative flag.

Reply:
Accidentally removed, I returned it back there.